draft-ietf-httpbis-retrofit.md

```diff
@@ -380,4 +384,39 @@ Finally, add a new column to the "Cookie Attribute Registry" established by {{CO
 --- back
 
 
+# The Date Structured Type {#date-type}
+
+This section defines a new Structured Fields data type, Date.
+
+Dates have a data model that is similar to Integers, representing a (possibly negative) delta in seconds from January 1, 1970 00:00:00 UTC, excluding leap seconds.
```

Comment:
Should this definition (converting to an actual date) be embedded in the serializing/parsing instructions, such that the output is actually a date?  Or perhaps just a note that converting an integer to a date representation will be implementation-dependent.

Reply:
That would require us to define / choose a model for dates. I'd rather just stay with delta to epoch - it's widely used and understood.